alfresco-transform-tika/alfresco-transform-tika-boot/src/test/java/org/alfresco/transformer/TikaMetadataExtractsIT.java

```diff
@@ -104,8 +104,8 @@ public void testTransformation(TestFileInfo testFileInfo)
                 testFile(MIMETYPE_IMAGE_JPEG, "jpg", "quickIPTC-EXT.jpg"),
                 testFile(MIMETYPE_IMAGE_JPEG, "jpg", "quickIPTC-multi-creator.jpg"),
                 testFile(MIMETYPE_IMAGE_JPEG, "jpg", "testJPEG_IPTC_EXT.jpg"),
-                testFile(MIMETYPE_IMAGE_GIF, "gif", "quick.gif"),
-                testFile(MIMETYPE_IMAGE_PNG, "png", "quick.png"),
+                testFile(MIMETYPE_IMAGE_GIF, "gif", "quickIPTC.gif"),
+                testFile(MIMETYPE_IMAGE_PNG, "png", "quickIPTC.png"),
```

Comment:
These should be in addition to the original quick files, not instead.

Reply:
I did not remove the original tests. See line [295 ](https://github.com/Alfresco/alfresco-transform-core/blob/ad767d21a24cc5575f8ed27589ebaee53d4afe85/alfresco-transform-tika/alfresco-transform-tika-boot/src/test/java/org/alfresco/transformer/TikaMetadataExtractsIT.java#L295)and [412](https://github.com/Alfresco/alfresco-transform-core/blob/ad767d21a24cc5575f8ed27589ebaee53d4afe85/alfresco-transform-tika/alfresco-transform-tika-boot/src/test/java/org/alfresco/transformer/TikaMetadataExtractsIT.java#L412)

Reply:
Great thanks 👍  Looked a bit strange in the dif :)